pkgs/applications/networking/instant-messengers/discord/darwin.nix

```diff
@@ -116,5 +117,20 @@ stdenv.mkDerivation {
         lib.optionalString (!stdenv.buildPlatform.isDarwin) "pkgsCross.aarch64-darwin."
       }${pname} "$version" --file=./pkgs/applications/networking/instant-messengers/discord/default.nix --version-key=${branch}
     '';
+
+    tests = {
+      withVencord = self.override {
```

Comment:
This is invalid recursion. Just use finalAttrs

Reply:
It is invalid due to the way one of the fixpoints works

Reply:
I don't see how any of this is invalid.